### PR TITLE
Migrate into @oasisprotocol

### PR DESCRIPTION
**Pull request by lukaw3d**
https://www.npmjs.com/package/deoxysii is owned by willscott
so we can't publish https://github.com/oasislabs/deoxysii.js/pull/30

### CODE REVIEW

**Comment by peterjgilbert** [comment]
> Should we actually consider moving this over to oasisprotocol like the other Deoxys-II implementations we have (Rust, Go)?

Yes, we can do that

**Comment by kostko** [comment]
> > Should we actually consider moving this over to oasisprotocol like the other Deoxys-II implementations we have (Rust, Go)?
> 
> Yes, we can do that

Ok, I've moved the repository. @lukaw3d can you update this PR to change the package name to `@oasisprotocol/deoxysii`? Also I've invited you to the npm org.

**Comment by lukaw3d** [comment]
Updated

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,4 +1,2 @@
 
-# Only the creator of the repo and the oasislabs admins
-# can manage the github settings of this repository
-.github/ @Yawning @willscott @oasislabs/admin
+.github/ @kostko @peterjgilbert
```
Comment (kostko) [inline .github/CODEOWNERS:2]: Should probably be:
```suggestion
* @kostko @peterjgilbert
```

**Review by kostko — APPROVED** [review]

**Pull request merged**